src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -142,6 +142,13 @@ function toggle(a, recursive) {
         return false;
     }
 
+    if (doc.createEvent) {
+        var event = doc.createEvent('Event');
+        event.initEvent('sf-dump-expanded' === newClass ? 'sfbeforedumpexpand' : 'sfbeforedumpcollapse', true, false);
+    }
+
+    s.dispatchEvent(event);
```

Comment:
should be inside the "if"?

Reply:
Oops, indeed... Also added to the `if` condition.